### PR TITLE
Create Popup window

### PR DESCRIPTION
**Pull request by Zeph180**
- Create pop window
- Design mobile popup window
- Design desktop popup window
- Implement javascript for opening popup
- Implement javascript for closing popup

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @Zeph180, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have linters passing. Correct linters setup, you should have [html-css-js linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for this project. Also, no commits from your partner as it's a pair programming project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: js/Popup.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    projectDescExt: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+
+  },
+];
+
+function breakTitle(title, element) {
+  const index = title.indexOf(' ');
+  const strOne = title.substr(0, index);
+  const strTwo = title.substr(index + 1);
+  const formatTitle = `${strOne}<br /> ${strTwo}`;
```
Comment (fobadara) [inline js/Popup.js:30]: [REQUIRED] Kindly fix the images for your projects section and popup section.
image

image

**Review by fobadara — APPROVED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/Popup.js (hunk 1 above)
Comment (Zeph180) [reply js/Popup.js:30]: @fobadara 
Please remind me of what I'm supposed to fix because the comment seems to have no issue description. Thank you

Comment (Zeph180) [reply js/Popup.js:30]: @fobadara 
I appreciate any suggestion on what exactly I'm supposed to do

**Pull request merged**